views/healthcheck/index.js

```diff
@@ -3,6 +3,8 @@ var clone = require('clone');
 var _config = require('../../config');
 var config = clone(_config);
 var packageJSON = clone(config.package);
+var i18n = require('../../lib/i18n.js');
+var langmap = require('langmap');
```

Comment:
There seems to be a mismatch with case right now from langmap v.s. our default locale -- let's discuss